map/field/field.tscn

```diff
@@ -4,6 +4,7 @@
 [ext_resource type="Script" path="res://map/field/field.gd" id="2_snbne"]
 
 [node name="Field" type="Button"]
+z_index = -1
```

Comment:
Not sure why the z index was changed, I do not see any reason for it as this is a local change anyways.

Reply:
I was trying to fix the visibilty with the z_index at first, I probalby forgot to remove that change

Reply:
I figured, no need to apologize just change it